saleor/graphql/app/schema.py

```diff
@@ -55,9 +58,11 @@ def resolve_apps_installations(self, info, **kwargs):
     def resolve_apps(self, info, **kwargs):
         return resolve_apps(info, **kwargs)
 
-    @permission_required(AppPermission.MANAGE_APPS)
-    def resolve_app(self, info, id):
-        return graphene.Node.get_node_from_global_id(info, id, App)
+    def resolve_app(self, info, id=None):
+        app = info.context.app
+        if not id and app:
+            return app if app else None
```

Comment:
could be simplified to `return app or None` or simply `return app` assuming this can only be either app or None.